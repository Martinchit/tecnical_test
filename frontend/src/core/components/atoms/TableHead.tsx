import styled from 'styled-components';

export const TableHead = styled.th`
  text-align: center;
`;
