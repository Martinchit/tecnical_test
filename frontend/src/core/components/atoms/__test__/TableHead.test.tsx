import React from 'react';
import { shallow } from 'enzyme';

import { TableHead } from '../TableHead';

describe('TableHead', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<TableHead />);
    expect(wrapper).toMatchSnapshot();
  });
});
