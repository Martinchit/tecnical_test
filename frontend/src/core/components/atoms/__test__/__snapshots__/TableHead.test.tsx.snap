// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHead renders correctly 1`] = `
<th
  className="sc-AxjAm jWlVxK"
/>
`;
